pages/[id]/delegate.tsx

```diff
@@ -149,18 +151,20 @@ const Delegate: NextPage = () => {
                   <div className="flex align-center p-4 gap-1">
                     <div className="self-center font-mono">
                       {mounted && delegatee !== ZERO_ADDRESS ? (
-                        <div className="flex items-start gap-1">
-                          {delegateEnsAvatar && (
+                        <div className="flex items-center gap-1">
+                          {delegateEnsAvatar ? (
                             <img
                               className="inline-block h-6 w-6 rounded-full"
                               src={delegateEnsAvatar}
                               alt=""
                             />
+                          ) : (
+                            <Jazzicon diameter={24} seed={jsNumberForAddress(delegatee || '')} />
                           )}
-                          {delegateEnsName || delegatee}{' '}
+                          <p className="text-sm self-end">{delegateEnsName || delegatee}</p>
                           <InformationCircleIcon
                             id="delegate-address-information"
-                            className="h-4"
+                            className="h-4 self-start"
```

Comment:
```suggestion
                            className="h-4"
```

screenshot shows the info icon is no longer vertically aligned w the text; removing this is the fix. curious, did you not want the icon centered vertically anymore?

Reply:
Yes, good question. I thought it looked better higher, but if you guys think it looks worse I can change it back

Reply:
my preference is centered vertically -- or next to the "Delgated to" rather than the address

Reply:
https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/174/commits/a3d0c8939fa271e82b45d455ee75e5c8539798b1